tests/blackbox/stratis_cli_cert.py

```diff
@@ -37,7 +37,11 @@ def make_test_pool(pool_disks):
     (return_code, _, stderr) = exec_test_command(
         [STRATIS_CLI, "pool", "create", pool_name] + pool_disks
     )
-    assert return_code == 0, "return_code: %s, stderr: %s" % (return_code, stderr)
+    if return_code != 0:
+        raise RuntimeError(
+            "Expected return code of 0; actual return_code: %s, %s"
+            % (return_code, stderr)
+        )
```

Comment:
Is it worth factoring these blocks into a method? It looks like it shows up a few places and it's just barely more than trivial so it might be worth keeping that consistent in a method.

Reply:
We might as well abstract for stratis_cli_cert.py as well, while we're here, even though these checks are smaller...

Reply:
Resolved.